### PR TITLE
Add PangoFc-1.0.gir

### PR DESCRIPTION
**Pull request by velyan**
https://github.com/gtk-rs/pangocairo/issues/56

### CODE REVIEW

**Comment by EPashkin** [comment]
@velyan Sorry, to include in our repo this PR need update to dl.sh too
Note that you can use your fork in this state in your crate without any problem.

File: dl.sh [hunk 1]
```diff
@@ -14,6 +13,6 @@ VER="eoan"
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libjavascriptcoregtk-4.0-dev/download
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libsoup2.4-dev/download
 ./gir-dl.sh https://packages.ubuntu.com/$VER/amd64/libwebkit2gtk-4.0-dev/download
-
+./gir-dl.sh https://packages.debian.org/experimental/amd64/libpango1.0-dev/download
```
Comment (velyan) [inline dl.sh]: I'm not sure if this is correct 😳
@EPashkin wdyt?

Comment (EPashkin) [reply dl.sh]: It need something like  `ftp.de.debian.org/debian` in end to select mirror.
But this also replaces other "good" Pango-1.0.gir with "bad" one and Gir don't work with it.
I don't know how to fix it in right way: too many changes.
So I still suggest you try use your fork as is and ask question on generation if needed.

Comment (velyan) [reply dl.sh]: I see I'll close the PR for now and revert the change to this file then. Thanks!

**Pull request closed without merging**